app/modules/transcribe/subjects.factory.js

```diff
@@ -161,7 +153,7 @@ function SubjectsFactory($q, AnnotationsFactory, localStorageService, zooAPI, zo
     }
 
     function _setCurrent() {
-        _data.current = _queue.shift();
+        _data.current = _queue[Math.floor(Math.random() * _queue.length)];
```

Comment:
`Array.shift()` removes the first element from an array, but the new code here doesn't change the `_queue` array. Is that intentional? It looks like you will always serve the same small group of subjects to the volunteers with this code.

Reply:
Ah I just missed that, thanks! I'll push a fix

Reply:
@eatyourgreens please check last commit